packages/dds/cell/src/cell.ts

```diff
@@ -37,80 +37,41 @@ interface IDeleteCellOperation {
 }
 
 interface ICellValue {
-    // The actual value contained in the cell which needs to be wrapped to handle undefined
-    value: any;
+    /**
+     * The actual value contained in the `Cell`, which needs to be wrapped to handle `undefined`.
+     */
+    value: unknown;
 }
 
 const snapshotFileName = "header";
 
 /**
- * The SharedCell distributed data structure can be used to store a single serializable value.
```

Comment:
This comment was moved to the interface, since that is what most customers will see first.